src/Convert.java

```diff
@@ -0,0 +1,5 @@
+public class Convert {

```

Comment:
According to java code convention it's better use nouns as class name and verbs as method names
(Ex.: class - Convertor, method - toConvert)